pkg/client/caddy.go

```diff
@@ -37,6 +37,9 @@ func (cli *Client) NewCaddyDeployment(image string, placement api.Placement) (*d
 		Container: api.ContainerSpec{
 			Command: []string{"caddy", "run", "-c", "/config/caddy.json", "--watch"},
 			Image:   image,
+			LogDriver: &api.LogDriver{
+				Name: "local",
+			},
```

Comment:
What do you think about setting this `local` driver by default in `CreateServiceContainer` in the daemon?
Here:
https://github.com/psviderski/uncloud/blob/main/internal/machine/docker/server.go#L530-L535

Basically allow the user to set it explicitly in the spec if they want to. But if it's not set, then set it to `local` instead of using the default from Docker. This way we won't depend on the Docker daemon configuration at all.